src/main/java/com/jcabi/github/RtStatus.java

```diff
@@ -40,7 +43,6 @@
  * @version $Id$
  * @since 0.23
  */
-@Immutable
```

Comment:
@yegor256 Can you please elaborate? You mean because the generic T might be mutable?